doc/users/next_whats_new/streamplot_integration_control.rst

```diff
@@ -0,0 +1,17 @@
+Streamplot integration control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Two new options have been added to the `~.axes.Axes.streamplot` function that
+give the user better control of the streamline integration. The first is called
+``integration_max_step`` and multiplies the default max step computed by the
+integrator. The second is called ``integration_max_error`` and multiplies the
+default max error set by the integrator. Values for these parameters between
+zero and one reduce (tighten) the max step or error to improve streamline
+accuracy by performing more computation. Values greater than one increase
+(loosen) the max step or error to reduce computation time at the cost of lower
+streamline accuracy.
+
+The integrator defaults are both hand-tuned values and may not be applicable to
+all cases, so this allows the user to customize the behavior to specific use cases.
+Modifying only ``integration_max_step`` has proved effective, but it may be useful
```

Comment:
```suggestion
all cases, so this allows customizing the behavior to specific use cases.
Modifying only ``integration_max_step_scale`` has proved effective, but it may be useful
```
though I'm not really sure the second sentence here is beneficial.